openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/integer.hpp

```diff
@@ -17,16 +17,17 @@
 
 #include <boost/lexical_cast.hpp>
 #include <openscenario_interpreter/error.hpp>
-#include <std_msgs/msg/int64.hpp>
 #include <string>
 
 namespace openscenario_interpreter
 {
 inline namespace syntax
 {
-struct Integer : public std_msgs::msg::Int64
+struct Integer
 {
-  using value_type = decltype(std_msgs::msg::Int64::data);
+  using value_type = std::int64_t;
+
+  value_type data = 0;
```

Comment:
From N4659 11.6 (6.1)

> To zero-initialize an object or reference of type T means:
> if T is a scalar type (6.9), the object is initialized to the value obtained by converting the integer literal
0 (zero) to T

therefore the following code is sufficient:

```suggestion
  value_type data;
```